CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Do not scale product thumbnail images [#1094](https://github.com/bigcommerce/cornerstone/pull/1094)
 - Lazy load carousel images [#1090](https://github.com/bigcommerce/cornerstone/pull/1090)
 - Theme Editor menu item updates for ease of use [#1091] (https://github.com/bigcommerce/cornerstone/pull/1091)
+- Upgrades all dependencies except for Foundation and jQuery [#1069](https://github.com/bigcommerce/cornerstone/pull/1069)
```

Comment:
@mjschock can you fix the line above & remove the space between `[#1091] (https://github.com/bigcommerce/cornerstone/pull/1091)` 🙏

Reply:
@junedkazi 👍